airflow/models/taskinstance.py

```diff
@@ -374,6 +377,780 @@ def _creator_note(val):
         return TaskInstanceNote(*val)
 
 
+def _execute_task(task_instance, context, task_orig):
```

Comment:
Hard to review this move but I think we have no other choice but to trust the tests that all is moved 1-1